cccs-build/superset/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM uchimera.azurecr.io/cccs/superset-base:feature_CLDN-867
```

Comment:
This line will be updated in a subsequent PR (once there is an image we can point to with a tag from our release branch).